### PR TITLE
Add Minimal Angular Template

### LINKED ISSUE #7199: Create Sample Project for Angular + Web3.js
Create a sample project that demonstrates building a simple dApp with Angular and Web3.js. Create a usage guide that explains the steps that are necessary to use Web3.js with Angular.

### LINKED ISSUE #7213: Create Angular template documentation
2024 H2 Action Item

### PR DESCRIPTION
**Pull request by danforbes**
Closes web3/web3.js#7199
Closes web3/web3.js#7213

### CODE REVIEW

File: templates/min/web3js-angular-dapp-min/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "version": "0.1.0",
+  "description": "Minimal template for bootstrapping a Web3.js + Angular dApp",
+  "contributors": [
+    "ChainSafe <info@chainsafe.io>"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
```
Comment (jdevcs) [inline templates/min/web3js-angular-dapp-min/package.json:12]: For this template what do you think for adding `build` (and later `test` ) in CLI tool's github CI, so with every PR we will be able to detect if any thing is breaking.

**Review by jdevcs — COMMENTED** [review]
also for using this template with cli tool could you modify [create-web3js-dapp](https://github.com/web3/create-web3js-dapp/tree/main)/[src](https://github.com/web3/create-web3js-dapp/tree/main/src) 
/index.js in same PR? so its usable with 

- `npx create-web3js-dapp angular` , 
- `npx create-web3js-dapp angular min` and 
- also with interactive cli tool screen. thanks

**Comment by danforbes** [comment]
> also for using this template with cli tool could you modify [create-web3js-dapp](https://github.com/web3/create-web3js-dapp/tree/main)/[src](https://github.com/web3/create-web3js-dapp/tree/main/src) /index.js in same PR? so its usable with
> 
> * `npx create-web3js-dapp angular` ,
> * `npx create-web3js-dapp angular min` and
> * also with interactive cli tool screen. thanks

Can we create a separate Issue for this under the CLI tool Epic we talked about? I'm trying to keep this PR scoped to web3/web3.js#7199 and web3/web3.js#7213.

File: templates/min/web3js-angular-dapp-min/package.json (hunk 1 above)
Comment (danforbes) [reply templates/min/web3js-angular-dapp-min/package.json:12]: Great idea! I will add that 💯

**Comment by danforbes** [comment]
> also for using this template with cli tool could you modify [create-web3js-dapp](https://github.com/web3/create-web3js-dapp/tree/main)/[src](https://github.com/web3/create-web3js-dapp/tree/main/src) /index.js in same PR? so its usable with
> 
> * `npx create-web3js-dapp angular` ,
> * `npx create-web3js-dapp angular min` and
> * also with interactive cli tool screen. thanks

@jdevcs I've created [a draft PR for the above](https://github.com/web3/create-web3js-dapp/pull/5). It's downstream from the branch for this PR so I will move it out of draft state once this PR has been merged.

**Comment by jdevcs** [comment]
> > also for using this template with cli tool could you modify [create-web3js-dapp](https://github.com/web3/create-web3js-dapp/tree/main)/[src](https://github.com/web3/create-web3js-dapp/tree/main/src) /index.js in same PR? so its usable with
> > 
> > * `npx create-web3js-dapp angular` ,
> > * `npx create-web3js-dapp angular min` and
> > * also with interactive cli tool screen. thanks
> 
> @jdevcs I've created [a draft PR for the above](https://github.com/web3/create-web3js-dapp/pull/5). It's downstream from the branch for this PR so I will move it out of draft state once this PR has been merged.

plus also create issue for this and add in sprint (cc: @mconnelly8  ) thanks

**Comment by danforbes** [comment]
> plus also create issue for this and add in sprint (cc: @mconnelly8 ) thanks

Created #6

**Review by jdevcs — APPROVED** [review]

**Pull request merged**